utopia-remix/app/models/projectAccess.server.ts

```diff
@@ -29,6 +29,16 @@ export async function createProjectAccess(params: {
   projectId: string
   accessLevel: AccessLevel
 }): Promise<void> {
+  // check if access level already exists
+  const projectAccess = await prisma.projectAccess.findUnique({
```

Comment:
this should be / have been put inside the transaction below. Moreover with this check in place (if this is the implementation way we want to go for), the upsert is unnecessary and can just be a `create` when `projectAccess == null`.

Reply:
good point, changed in #5156